docs/reference/gettingstartedexample/gse-debug-guide.md

```diff
@@ -0,0 +1,242 @@
+- TODO: Add link to the 'serial channel status' instructions below
+ 
+If you cannot figure out what part of the serial output data path is causing the timing errors, or are seeing FIFO overflow errors, take these steps to reduce system bandwidth. You can start by refering to gse-gen-common.md#reducing-system-bandwidth-usage. Afterwards, try some of these tasks to see which part of your system or data set is responsible for causing overflows.
+
+TODO: maybe flesh these steps out a bit more? They are really just ways to get started...
+- Disable display (or shrink image size)
+- Use smaller packets (reduce bandwidth requirements)
+- Set your timing error limit so high that we don't error out, and then see if DRAM manager still fills up
+- Use the /gse-create-tdms.md to create a known good set of csi-2 packet data with the desired timing characteristics and verify your generation completes successfully 
+- Reduce serial output channel count
+
+### How to debug Acquisition FIFO overflow
+The first thing to check is the Serial Input Channel Status indicator. You should verify that the bytes going in and coming out of the DRAM match. If the DRAM manager is overflowing, it likely means the FPGA to Host FIFO is pushing back on the DRAM manager and the DRAM partition is getting full.
+- TODO: Add link to the 'serial channel status' instructions below
+
+It is hard to distinguish if DMA is pushing back on DRAM causing DRAM overflow (e.g. system can't receive data fast enough, this can be many root causes), or if it's just data coming in faster than we can handle (FPGA simply can't handle this data rate).
+
+- If downstream is not ready for output and you overflow, likely it's downstream at issue (DMA pushing back)
+- If downstream is ready, and you overflow, DRAM manager might not be able to handle. This scenario is generally unlikely based on empirical testing. 
+
+If one of the two scenarios above is suspected, you can instrument the FPGA to determine where in the data path you are getting clogged up.
+TODO - add link to FPGA instrumentation guide.
+
+Things we can suggest to look into:
+- Double check Active Lane rate and count are matched throughout your system (Camera -> Instrument)
+- Look at the Serial Channel Input Status array for patterns, are problems always following a specific channel?
+- Verify DRAM filling, try to compare rate of DRAM filling vs FIFO transfer rate to host
+- Instrument the datapath to look for specific hard to detect states? TODO - link to instrumentation part
+- Reduce the acquisition to a single channel and verify that it stops successfully.
+
+### How to debug image display that does not show images or has a low frame rate
+Images will not be displayed or will be displayed with a low frame rate when the system does not have the bandwidth to keep up with the images coming from the FPGA. 
```

Comment:
Suggested wording change:
"Images display with a reduced frame rate or do not display at all when the system does not have the bandwidth to keep up with the images coming from the FPGA."